### ISSUE TITLE
make information flow delays visible

### ISSUE THREAD

**Issue by scoid** [meine-stadt-transparent/meine-stadt-transparent#868]
There is a nuisance in many municipalities of short-term settings of documents in the council information systems.

If I had one wish, it would be the installation of the feature that the date of publication for the councils and citizen representatives is compared with the publication for the general population. This time difference should be mapped.

It needs here still another annotation possibility, since which date already a submission of the administration at all might be known and the first setting into the system around the blatant delays to make visible.

The short deadlines for the publication of documents lead to a qualitatively mutilated and shallow debate about topics.

**Comment by scoid** [comment]
If we want more transparency, it would be good to know how long an administration has been squatting on information since it arrived in the mail, until it is posted in the council information system. Here one must create upstream in the constitutions such conditions. Maybe in oparl it needs another variable for this?

Then comes the actual meeting.

The third date is the announcement for the general public.

**Comment by konstin** [comment]
I totally agree that this would be an important information to have! The question for me is, is the arrived-in-the-mail date is tracked anywhere? If nobody enters this information into their RIS backend in a standardized fashion there's no way to export it through oparl and subsequently to display

**Comment by scoid** [comment]
> I totally agree that this would be an important information to have! The question for me is, is the arrived-in-the-mail date is tracked anywhere? If nobody enters this information into their RIS backend in a standardized fashion there's no way to export it through oparl and subsequently to display

I make an issue from this and start asking my councils City and Landkreis to implement it in its constitution and of course after the Lower Saxony election the new government to do the same. This country is more in my reach.
It must be tracked somewhere (I hope), otherwise we could not insist on a time limit for answering, questions to authorities.

For a intermediate time period it could be added manually to each document, until it gets into a council information system.

**Comment by scoid** [comment]
> > I totally agree that this would be an important information to have! The question for me is, is the arrived-in-the-mail date is tracked anywhere? If nobody enters this information into their RIS backend in a standardized fashion there's no way to export it through oparl and subsequently to display
> 
> I make an issue from this and start asking my councils City and Landkreis to implement it in its constitution and of course after the Lower Saxony election the new government to do the same. This country is more in my reach. It must be tracked somewhere (I hope), otherwise we could not insist on a time limit for answering, questions to authorities.
> 
> For a intermediate time period it could be added manually to each document, until it gets into a council information system.

This date tag, should be applied for ANY document in the RIS and it should be an open defined API for this in the DMS document information system of the local authority.
